lib/coprl/presenters/plugins/open_id/components/helpers.rb

```diff
@@ -0,0 +1,28 @@
+module Coprl
+  module Presenters
+    module Plugins
+      module OpenId
+        module Helpers
+          def scope_string(scope)
+            case scope
+            when String
+              scope
+            when Array
+              scope.map(&:to_s).join(" ")
+            else
+              raise ArgumentError, "Scope must be a string or an array"
+            end
+          end
+
+          def html_safe(string)
+            string.respond_to?(:html_safe) ? string.html_safe : string
+          end
```

Comment:
We've used this a few times now. Should this be incorporated into COPRL itself?

Reply:
It's already in the `hotwire-spike` version of COPRL. Might be worth pulling in.

Reply:
thinking out loud: do we need to pull in anything? will the plugin just get this for free when running with COPRL@hotwire-spike? maybe this method can just be removed from here.

Reply:
Yes, but it looks like you need this now so I was thinking it might be worth pulling into COPRL before we merge the `hotwire-spike` branch.

Reply:
I think the simplest path is to just merge this as-is and rip out this plugin's `html_safe` method at some point after `html_safe` lands in COPRL `main`. we can still back-port `html_safe` to mainline COPRL now (for other plugins), but this PR doesn't need to wait for/depend on that.

Reply:
added!